spec/tbd_tests_spec.rb

```diff
@@ -5246,21 +5868,27 @@
         t = edge[:type]
         expect(types.values).to include(t)
 
-        n_transitions += 1 if edge[:type] == types[:t1]
-        n_parapets    += 1 if edge[:type] == types[:t2]
-        n_fen_edges   += 1 if edge[:type] == types[:t3]
-        n_heads       += 1 if edge[:type] == types[:t4]
-        n_sills       += 1 if edge[:type] == types[:t5]
-        n_jambs       += 1 if edge[:type] == types[:t6]
+        n_transitions   += 1 if edge[:type] == types[:t1]
+        n_parapets      += 1 if edge[:type] == types[:t2]
+        n_fen_edges     += 1 if edge[:type] == types[:t3]
+        n_heads         += 1 if edge[:type] == types[:t4]
+        n_sills         += 1 if edge[:type] == types[:t5]
+        n_jambs         += 1 if edge[:type] == types[:t6]
+        n_skylightheads += 1 if edge[:type] == types[:t7]
+        n_skylightsills += 1 if edge[:type] == types[:t8]
+        n_skylightjambs += 1 if edge[:type] == types[:t9]
```

Comment:
A significant number of _RSpecs_ needed updating following the 90.1 adaptations, namely concerning subsurface perimeter edges. Not going to comment every fix; this is one representative example.